docs/pages/build-reference/limitations.mdx

```diff
@@ -12,7 +12,7 @@ EAS Build is designed to work for any React Native project. However, it is good
 
 <Collapsible summary="Fixed memory and CPU limits on build worker servers">
 
-The [Server infrastructure reference](/infrastructure) contains the most up-to-date information about the current specifications of the Android (Ubuntu) and iOS (macOS) build servers. You may find that the resources available are not sufficient to build your app if your build process requires more than 12GB RAM. In the future we will be adding more powerful configurations to increase memory limits and speed up build times.
+The [Server infrastructure reference](./infrastructure) contains the most up-to-date information about the current specifications of the Android (Ubuntu) and iOS (macOS) build servers. You may find that the resources available are not sufficient to build your app if your build process requires more than 12GB RAM. In the future we will be adding more powerful configurations to increase memory limits and speed up build times.
```

Comment:
```suggestion
The [Server infrastructure reference](/build-reference/infrastructure/) contains the most up-to-date information about the current specifications of the Android (Ubuntu) and iOS (macOS) build servers. You may find that the resources available are not sufficient to build your app if your build process requires more than 12GB RAM. In the future, we will add more powerful configurations to increase memory limits and speed up build times.
```

A small suggestion is to include the section path to in the URL. Also, a small typo fix (which is unrelated to the changes in this PR).

Reply:
Thanks for the heads up!
If absolute paths are the convention for links in the docs should we go ahead and fix the other link in this file too?
https://github.com/expo/expo/blob/e515643f4bdd1ed45ec1625055b5bfdc61b9b624/docs/pages/build-reference/limitations.mdx?plain=1#L25

Reply:
Yes 😄